scania-digital-design-system/sdds-website
# AB#810 Update image URL with Scania AWS cloudfront

**Describe pull-request**  
- Update Image URL in CMS that was still using external S3 to point to Scania CF

**Solving issue**  

Fixes: [AB#810](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/810)

**How to test**  
- Check in netlify preview if images are all good.
- See this PR changes to check which images affected (most of them are images within table, or code example) => Spacing page, colour specification, card code example.
